Remotion/Web/Development.WebTesting/Accessibility/AccessibilityAnalyzer.cs

```diff
@@ -42,7 +42,7 @@ public class AccessibilityAnalyzer
     /// </summary>
     [NotNull]
     public static AccessibilityAnalyzer CreateForRemoteWebDriver (
-        [NotNull] RemoteWebDriver remoteWebDriver,
+        [NotNull] OpenQA.Selenium.WebDriver remoteWebDriver,
```

Comment:
Use IWebDriver instead, update parameter name, method name, and xmldocs.

Create breaking change ticket for the rename.

Also keep the original method with an an obsolete-attribute. Please check other usages of ObsoleteAttribute for how this should look like.